app/views/notify_user/note_added.text.haml

```diff
@@ -0,0 +1,3 @@
+A note was added for the following client:
+
+= client_notes_url(@client)
```

Comment:
This will need to link to the window version of notes, since most people don't have access to the client section.